### PR TITLE
MockTest for CompositeFeedLogger class is ready

### PR DESCRIPTION
**Pull request by jsNickOs**

### CODE REVIEW

File: ss_admin_console/main/default/classes/poc/logger/CompositeFeedLogger.cls [hunk 1]
```diff
@@ -7,8 +7,8 @@ public with sharing class CompositeFeedLogger implements FeedLogger {
     }
 
     public void error(String messageError) {
-        for (FeedLogger feedLogger : feedLoggers) {
-            feedLogger.error(messageError);
+        for (FeedLogger logger : feedLoggers) {
```
Comment (mykhaylo-partyka) [inline ss_admin_console/main/default/classes/poc/logger/CompositeFeedLogger.cls:10]: Revert it back

File: ss_admin_console/main/tests/classes/CompositeFeedLoggerTest.cls [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    static void testLog() {
+        fflib_ApexMocks mocks = new fflib_ApexMocks();
+        fflib_ApexMocksConfig.HasIndependentMocks = true;
+        FeedLogger logger1 = (FeedLogger) mocks.mock(FeedLogger.class);
+        FeedLogger logger2 = (FeedLogger) mocks.mock(FeedLogger.class);
+        FeedLogger logger = new CompositeFeedLogger(new List<FeedLogger>{logger1, logger2});
+        Test.startTest();
+        logger.log('logTestMsg', '', '');
+        Test.stopTest();
+        ((FeedLogger) mocks.verify(logger1, 2)).log('logTestMsg', '', '');
```
Comment (mykhaylo-partyka) [inline ss_admin_console/main/tests/classes/CompositeFeedLoggerTest.cls]: 2?

**Review by mykhaylo-partyka — CHANGES_REQUESTED** [review]

File: ss_admin_console/main/tests/classes/CompositeFeedLoggerTest.cls (hunk 2 above)
Comment (jsNickOs) [reply ss_admin_console/main/tests/classes/CompositeFeedLoggerTest.cls]: Fixet it already

File: ss_admin_console/main/default/classes/poc/logger/CompositeFeedLogger.cls (hunk 1 above)
Comment (jsNickOs) [reply ss_admin_console/main/default/classes/poc/logger/CompositeFeedLogger.cls]: Reverted it

**Review by mykhaylo-partyka — APPROVED** [review]

**Pull request merged**